src/pages/Selector/index.jsx

```diff
@@ -28,23 +36,26 @@ const Selector = ({ employees, templates }) => (
         options={mockDataForSelect.clients}
       />
       <Select
+        data-select="template"
         text="Template"
         isRequired
         name="selectComponentTemplate"
         isSearchable
         isDisabled={false}
         options={templates}
       />
-      <GenerateCVButton />
+      <GenerateCVButton data-button="generate-cv" />
     </div>
-    <Modal isOpen title="Preview Document" bodyText={mockDataForModal.bodyText}>
-      <CancelModalButton />
-      <DownloadModalButton />
-    </Modal>
+    {employeeCV.length > 0 && (
```

Comment:
Sorry, my bad here. I'd also add a check on employeeCV so it's `{employeeCV && employeeCV.length > 0 && (`